core/jvm/src/main/scala/cats/effect/unsafe/IORuntimeCompanionPlatform.scala

```diff
@@ -29,12 +29,14 @@ import javax.management.ObjectName
 
 private[unsafe] abstract class IORuntimeCompanionPlatform { this: IORuntime.type =>
 
+  private[this] val DefaultBlockerPrefix = "io-compute-blocker"
```

Comment:
```suggestion
  private[this] final val DefaultBlockerPrefix = "io-compute-blocker"
```

Then the compiler inlines it. In theory :)